monkey/monkey_island/cc/repository/i_agent_repository.py

```diff
@@ -40,6 +40,16 @@ def get_running_agents(self) -> Sequence[Agent]:
         :raises RetrievalError: If an error occurred while attempting to retrieve the `Agents`
         """
 
+    @abstractmethod
+    def get_progenitor(self, descendant: Agent) -> Agent:
+        """
+        Gets the `progenitor` Agent from the descendant.
```

Comment:
What does the \` do? It will appear like `this` in the documentation?

Reply:
Regardless, "progenitor" isn't a type, so I don't think it should appear like `this`.